NewSpring/corinth
# chore: Upgrade 1.5

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR upgrades the client of the app to core version 1.5.0.

### How do I test this PR?
First Things First: This shouldn't be tested until the API changes are tested and merged and in production. These changes wouldn't work without those changes and we can't push this to the app stores without the API in place to handle it.

So ... if the API changes are in production, you can run the app on this branch pointing to the production API. Again, just make sure that all the things work. There is nothing in this update that should be changing any functionality. 

Some things to keep an eye on:
* There maybe some places where core changed text sizes or boldness. 
* You may see some improved image loading times. 
* Check to make sure that if you delete the app that it takes you through onboarding but if you sign out of the app and sign back in again that it doesn't. 

You can use this doc as a guide for testing: https://docs.google.com/document/d/1JoBErzORE2pMSfYS-79dw4O31RxKpMmE4vJPTNmU8uc/edit

Also, here is a list of all the 1.5.0 changes: https://docs.google.com/document/d/13_roIFoqiX2_gwQ6OvSnnwFT1cknxYWiDctHeggDbZI/edit#

**Note**: Please check that videos work. I am having issues getting them to work correctly in my sim, but it's not just the upgrade that's causing the problem. I couldn't get them to work on `master` either ... but they are working in the app so ...

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
